backend/.env.example

```
@@ -1,2 +1,14 @@
-OPENAI_API_KEY=your_openai_api_key_here
-DATABASE_URL=postgresql://docuchat:docuchat_pass@postgres:5432/docuchat
+DATABASE_URL=postgresql+asyncpg://user:pass@db:5432/docuchat
+REDIS_URL=redis://redis:6379/0
+OPENAI_API_KEY=sk-***
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The OPENAI_API_KEY key should go before the REDIS_URL key

(UnorderedKey)